start/routes.ts

```diff
@@ -98,18 +98,20 @@ router
     router.get("/about_us", [AboutUsController, "index"]);
 
     router.get("/newsfeed/latest", [NewsfeedController, "latest"]);
-    router.group(() => {
-      router.get("/cache_reference_number", [
-        CacheReferenceNumberController,
-        "index",
-      ]);
-      router
-        .patch("/cache_reference_number/bump", [
-          CacheReferenceNumberController,
-          "bump",
-        ])
-        .use(middleware.auth());
-    });
+    router
+      .group(() => {
+        router.get("/", [MobileConfigController, "index"]);
+        router
+          .patch("/bump", [MobileConfigController, "bump"])
+          .use(middleware.auth());
+        router
+          .patch("/update_lookahead/:new_value", [
+            MobileConfigController,
+            "updateCount",
+          ])
```

Comment:
hell nah
that's now how you make rest endpoints
since the mobile config singleton will likely get more random fields, just define a generic update method that can update any of the mobile config fields - you can even use the base controller class and just not use the other methods it generates